src/common/mem.c

```diff
@@ -43,18 +43,18 @@ enum {
 static int daos_md_backend = DAOS_MD_PMEM;
 
 /** Sets up global settings for the pmem objects.
+ *
+ *  \param	md_on_ssd[IN]	Indicate if MD-on-SSD mode is set.
  *
  *  \return	0 on success, non-zero on failure.
  */
 int
-umempobj_settings_init(void)
+umempobj_settings_init(int md_mode)
```

Comment:
The 'md_mode' could be changed as a 'bool' to specify if md-on-ssd is configured, and this function internally read an env var to see which allocator should be used (BMEM or ADMEM, BMEM is default) when md-on-ssd is configured.

Reply:
done, name changed to "DAOS_MD_ON_SSD_MODE"